tests/test_cookies.py

```diff
@@ -151,6 +157,7 @@ def test_cookie_expires_naive(client):
     assert cookie.value == 'bar'
     assert cookie.domain is None
     assert cookie.expires == datetime(year=2050, month=1, day=1)
+    assert cookie.same_site == 'Strict'
```

Comment:
Please also add a couple of tests to demonstrate case-insensitivity (i.e., that the framework does not require case-sensitive values be used when setting the attribute).